docs-source/content/release-notes.md

```diff
@@ -30,6 +30,27 @@ draft: false
 
 ### Change log
 
+#### Operator 3.1.0
+
+* Updated [default resource and Java options](faq/resource-settings.md) ([#1775](https://github.com/oracle/weblogic-kubernetes-operator/pull/1775)).
+* Introspection failures are logged to the operator's log ([#1787](https://github.com/oracle/weblogic-kubernetes-operator/pull/1787)).
+* Mirror introspector log to a rotating file in the log home ([#1827](https://github.com/oracle/weblogic-kubernetes-operator/pull/1827)).
+* Reflect introspector status to domain status ([#1832](https://github.com/oracle/weblogic-kubernetes-operator/pull/1832)).
+* Ensure operator detects pod state changes even when watch events are not delivered ([#1811](https://github.com/oracle/weblogic-kubernetes-operator/pull/1811)).
+* Support configurable model home ([#1828](https://github.com/oracle/weblogic-kubernetes-operator/pull/1828)).
+* [Namespace management enhancements](faq/namespace-management.md) ([#1860](https://github.com/oracle/weblogic-kubernetes-operator/pull/1860)).
+* Limit concurrent pod shutdown during cluster shrink ([#1892](https://github.com/oracle/weblogic-kubernetes-operator/pull/1892)).
```

Comment:
shutdown -> shut down (shut down (v), shutdown (n) )
cluster shrink -> odd wording ( I see that these statements come from JIRA titles, which are not always written well. Should we keep them with their original wording or fix them?)